pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_pantheon-user-interface.adoc

```diff
@@ -0,0 +1,28 @@
+[id='pantheon-user-interface']
+= Pantheon user interface
+
+
+The user interface is organized with a main work area, an upper toolbar with a `Login` button and a show/hide button for displaying the sidebar, and a tabbed sidebar with the following tabs:
+
+* `Modules` - Use this tab to create new document modules, search for, edit, or delete modules.
+** `Search` - Provides advanced search options, including keyword search, search by product name, product version, document module  type (procedure, concept, reference) and upload date.
+*** Select a module check box to delete the selected module.
+*** Select a module name to view the module status and details, and publish, unpublish, preview or view the module in the customer portal.
+*** A drop-down menu allows editing the metadata of a module.
+** `New Module` - Use this tab to create a new module title.
```

Comment:
@emarcusRH We'll be removing the 'New Module' tab prior to Pantheon 2 go-live.

Reply:
@benradey Hi Ben, 
- will the uploader script via the CLI still be valid? or is that also a temporary functionality during development? 
- If it is retained, I am guessing only an Admin level will be able to access it?

Reply:
@emarcusRH I actually just talked about the uploader script in another comment. Give this a read and let me know if it answers your questions:

https://github.com/redhataccess/pantheon/pull/203#discussion_r372536250